db/migrate/20210830080830_add_upstream_for_profiles.rb

```diff
@@ -0,0 +1,11 @@
+class AddUpstreamForProfiles < ActiveRecord::Migration[5.2]
+  def up
+    add_column :profiles, :upstream, :boolean
+    # Initially all canonical profiles are marked as upstream
+    Profile.canonical.update_all(upstream: true)
```

Comment:
Why not use a default value? If we update this field on non-canonical profiles, what issues would that cause?

Also: we may want an index on this column.

Reply:
It would cause semantical issues :see_no_evil: my eyes would not like it :confused: